docs/installation-docker.rst

```diff
@@ -78,21 +99,169 @@ minutes depending on the size of the data dump.
+    Post-processed 'css/zones.css' as 'css/zones.718d56a0cdc0.css'
+    Post-processed 'css/zones.css.map' as 'css/zones.css.6be0969a4847.map'
+
+    1717 static files copied to '/app/static', 1799 post-processed.
+
 Visit the Homepage
 ==================
 Open the homepage at http://localhost:8000 . You've installed Kuma!
 
-Coming Soon
-===========
-Docker support is experimental, so the full features of the Vagrant install are
-not supported.  The next steps are enabling and documenting how to:
+Create an admin user
+====================
+Many Kuma settings require access to the Django admin, including
+configuring social login.  It is useful to create an admin account with
+password access for local development.
+
+If you want to create a new admin account, use ``createsuperuser``::
+
+    docker exec -it kuma_web_1 ./manage.py createsuperuser
+
+This will prompt you for a username, email address (a fake address like
+``admin@example.com`` will work), and a password.
+
+If your database has an existing account that you want to use, use the Django
+shell, similar to this::
+
+    docker exec -it kuma_web_1 ./manage.py shell_plus
+    >>> me = User.objects.get(username='admin_username')
+    >>> me.set_password('mypassword')
+    >>> me.is_superuser = True
+    >>> me.is_staff = True
+    >>> me.save()
+    >>> exit()
+
+With a password-enabled admin account, you can log into Django admin at
+http://localhost:8000/admin/login/
+
+.. _Disable your admin password:
+
+When social accounts are enabled, the password can be disabled with the Django
+shell::
+
+    docker exec -it kuma_web_1 ./manage.py shell_plus
+    >>> me = User.objects.get(username='admin_username')
+    >>> me.set_unusable_password()
+    >>> me.save()
+    >>> exit()
+
+Enable the wiki
+===============
+By default, the wiki is disabled with a
+:doc:`feature toggle <feature-toggles>`.  To enable editing:
+
+#. Log in as an admin user.
+#. Open the `Waffle / Flags`_ section of the admin site.
+#. Click "`ADD FLAG`_", above the Filter sidebar.
+#. Enter "kumaediting" for the Name.
+#. Set "Everyone" to "Yes".
+#. Click "SAVE" at the bottom of the page.
+
+If you are using a populated database, the "kumaediting" flag may already
+exist.
+
+You can now visit http://localhost:8000/docs/new to create new wiki pages.
+
+Many contributors use a a personal page as a testing sandbox, with a title
+such as "User:myusername".
+
+.. _Waffle / Flags: http://localhost:8000/admin/waffle/flag/
+.. _ADD FLAG: http://localhost:8000/admin/waffle/flag/add/
+
+Enable KumaScript
+=================
+By default, `KumaScript`_ is disabled by the default timeout of `0.0` seconds.
+To enable KumaScript:
+
+#. Log in as the admin user.
+#. Open the `Constance / Config`_ section of the admin site.
+#. Change ``KUMASCRIPT_TIMEOUT`` to 600.
+#. Click "SAVE" at the bottom of the page.
+#. Import the `KumaScript auto-loaded modules`_:
+
+::
+
+   docker exec -it kuma_web_1 ./manage.py import_kumascript_modules
```

Comment:
This implies that the database initialization was incorrect, and you ended up with a latin1 encoding.  It succeeded the second time because it skips download of the half-initialized macro